### PR TITLE
Fix minor typos in oar

### PR DESCRIPTION
**Pull request by barboras7**

### CODE REVIEW

**Comment by barboras7** [comment]
@rioliu-rh Just some small fixes, please review once you're available :) Thanks

File: oar/cli/cmd_group.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def print_version(ctx, param, value):
     expose_value=False,
     is_eager=True,
 )
-@click.option("-r", "--release", help="z-stream releaes version")
+@click.option("-r", "--release", help="z-stream release version", required=True)
```
Comment (rioliu-rh) [inline oar/cli/cmd_group.py:41]: required flag is removed due to enhancement [OCPQE-25622](https://issues.redhat.com/browse/OCPQE-25622). validation will be done implicitly

File: oar/cli/cmd_image_consistency_check.py [hunk 2]
```diff
@@ -50,15 +50,15 @@ def image_consistency_check(ctx, build_number, for_nightly):
...
+                                "registry.ci.openshift.org/ocp/release: " + cs.get_candidate_builds().get("x86_64") 
                             )
                         else: 
                             raise JenkinsHelperException(
                                 f"no candidate nightly build for x86_64 architecture"
                                 )
                     else:
                         pull_spec = (
-                            "quay.io/openshift-release-dev/ocp-release:" + cs.release + "-x86_64" 
+                            "quay.io/openshift-release-dev/ocp-release: " + cs.release + "-x86_64"
```
Comment (rioliu-rh) [inline oar/cli/cmd_image_consistency_check.py:61]: space should not be there for image tag.
e.g. registry.ci.openshift.org/ocp/release:4.16.0-0.nightly-2024-09-16-095421

File: oar/cli/cmd_image_consistency_check.py [hunk 3]
```diff
@@ -50,15 +50,15 @@ def image_consistency_check(ctx, build_number, for_nightly):
                     if for_nightly:
                         if "x86_64" in cs.get_candidate_builds():
                             pull_spec = (
-                                "registry.ci.openshift.org/ocp/release:" + cs.get_candidate_builds().get("x86_64") 
+                                "registry.ci.openshift.org/ocp/release: " + cs.get_candidate_builds().get("x86_64")
```
Comment (rioliu-rh) [inline oar/cli/cmd_image_consistency_check.py:53]: same concern as below

File: oar/cli/cmd_image_consistency_check.py (hunk 2 above)
Comment (barboras7) [reply oar/cli/cmd_image_consistency_check.py]: Thanks Rio, I did this automatically and did not realize this. All issues should be fixed now

**Pull request merged**